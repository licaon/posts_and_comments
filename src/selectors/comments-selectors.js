export const getComments = state => state.CommentsReducer;
export const getCommentsState = state => state.CommentsStateReducer;
