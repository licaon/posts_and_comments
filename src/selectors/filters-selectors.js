export const getSearchFiltersValue = state => state.SearchReducer;
