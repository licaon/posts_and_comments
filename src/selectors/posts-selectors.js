import { createSelector } from 'reselect';

export const getPosts = state => state.PostsReducer.posts;

export const getFilteredPosts = createSelector(
  [getPosts],
  posts => posts,
);
