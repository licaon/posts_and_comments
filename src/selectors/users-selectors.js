export const getUsers = state => state.UsersReducer.users;
