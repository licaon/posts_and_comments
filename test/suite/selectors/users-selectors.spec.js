import * as UsersSelectors from 'src/selectors/users-selectors';

describe('UsersSelectors', () => {
  it(`should execute getUsers correctly`, (done) => {
    expect(UsersSelectors.getUsers({
      UsersReducer: {
        users: 'UsersReducer.users',
      },
    })).toMatchSnapshot();

    done();
  });
});
