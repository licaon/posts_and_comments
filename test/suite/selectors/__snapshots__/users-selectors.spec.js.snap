// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersSelectors should execute getUsers correctly 1`] = `"UsersReducer.users"`;
